packages/app/src/utils/validation.ts

```diff
@@ -0,0 +1,22 @@
+import { SupportedChain, SupportedChainId } from "../constants/chain"
+
+export const checkIsValidChain = (currentChain: number): boolean => {
```

Comment:
If we are not in a publication, it should return `true` as long as we are connected to one of the supported chains.

If not, it fails when creating a new publication,